distribution/scripts/jmeter/start-jmeter-test.sh

```diff
@@ -387,11 +396,15 @@ function test_scenarios() {
                     test_start_timestamp=$(date +%s)
                     echo "Start timestamp: $test_start_timestamp"
                     # Run JMeter in background
-                    $jmeter_command &
+                    var_path1=`pwd`
```

Comment:
The same question here. What is the directory path (value of `var_path1`) we are at when generating the payloads?